function.js

```diff
@@ -0,0 +1,105 @@
+// 1 Create a function that takes 3 arguments 
+function threeArg(a,b,c){
+    let d = a + b + c ;
+    return d;
+}
+
+var sum = threeArg(3,4,3)
+console.log(sum);
+
+// 2  function  colorCar
+function colorCar(letter ,color, auto){
+    let syn = letter +" " + color + " " + auto ;
+    return syn;
+}
+
+var clr = colorCar( "a","red","car");
+console.log(clr);
+
+// 3  a function that takes the object as a paramete
+
+function obj(objec){
+return objec;
+}
+
+let obje = {
+    name : "Abdul",
+    old : 30
+}
+
+let variable = Object.keys(obje);
+let value = Object.values(obje);
+
+for(let i = 0; i < variable.length; i++){
+    console.log(obj(variable[i]));
+    console.log(obj(value[i]));
+}
```

Comment:
Good solution, just a tip, there are more [ways](https://zellwk.com/blog/looping-through-js-objects/) to do this!